src/components/DeploymentDetails.vue

```diff
@@ -30,7 +31,18 @@
           <div v-for="automation in relatedAutomations" :key="automation.id" class="deployment-details-triggers__value">
             <AutomationIconText :automation-id="automation.id" />
           </div>
+
+          <!-- If the button has a :to set, it doesn't seem to respect the :disabled param -->
```

Comment:
This feels like a bug we can solve, opened a ticket [here](https://github.com/PrefectHQ/prefect-design/issues/1406)